Makefile

```diff
@@ -140,6 +151,14 @@ $(TARGET).bin: $(TARGET).elf
 flash:
 	$(SF) write $(TARGET).bin 0x8000000
 
+ventilation_test: test/testRunner.c src/application/ventilation.o test/ventilation-test.o test/stubs/systemInfo_stub.o test/stubs/valve_stub.o test/stubs/dss_stub.o test/testUtil.o
```

Comment:
I am not sure that explicitly listing the files here is the best strategy. It seems error prone if someone forgets to update the make file.

Can't we just build all of the tests at once? Or is there a reason to separate them? 

Basically, how can we clean this up?

Reply:
It would require to have all the stubs except the one for the function tested. and only the c file in the application that is the one tested.

Reply:
I will create an issue to do this in the future